node/full.go

```diff
@@ -255,7 +260,11 @@ func (n *FullNode) OnStart() error {
 	}
 	if n.conf.Aggregator {
 		n.Logger.Info("working in aggregator mode", "block time", n.conf.BlockTime)
-		go n.blockManager.AggregationLoop(n.ctx)
+		if n.conf.ProgressiveSequencer {
```

Comment:
instead of calling two separate functions, can you just pass the `n.conf.ProgressiveSequencer` flag to `AggregationLoop` and let it handle both cases?

Reply:
done